superdesk/media/media_editor.py

```diff
@@ -116,8 +116,9 @@ def create(self, docs):
             for operation, param in edit.items():
                 try:
                     out = self.transform(out, operation, param)
-                except ValueError as e:
-                    raise errors.SuperdeskApiError.badRequestError('invalid edit instructions: {msg}'.format(msg=e))
+                except ValueError:
+                    # if the operation can't be applied just ignore it
+                    None
```

Comment:
there is [pass](https://docs.python.org/3.2/reference/simple_stmts.html#the-pass-statement) for noop